.circleci/config.yml

```diff
@@ -204,9 +204,21 @@ jobs:
     steps:
       - attach_workspace: 
           at: .
-      - run:
+      - run: 
           name: 'Install and Test (check-failure)'
           command: 'install-and-test-ext check-failure'
+  test_check-pg-upgrade:
+    docker:
+      - image: 'heisenberg302/pgupgradetester:latest'
+    working_directory: /home/circleci/project     
+    steps:
+      - attach_workspace:
+          at: .
+      - run: 
+          name: 'Install and test postgres upgrade'
+          command: |
+            install-and-test-ext --target check-upgrade --old-pg-version 10 --new-pg-version 11
+            install-and-test-ext --target check-upgrade --old-pg-version 11 --new-pg-version 12
```

Comment:
can we please create to jobs in the matrix? This helps in attributing faults easier, and also allows for more parallelism.